metaseq/cli/train.py

```diff
@@ -144,10 +144,12 @@ def main(cfg: DictConfig) -> None:
             cfg.distributed_training,
             use_sharded_state=cfg.distributed_training.use_sharded_state,
         ):
-            model = fsdp_wrap(
-                task.build_model(cfg.model),
-                process_group=distributed_utils.get_data_parallel_group(),
-            )
+            model = task.build_model(cfg.model)
+            if not isinstance(model, FullyShardedDataParallel):
```

Comment:
Just to confirm, this is for loading up consolidated model for training?

Reply:
Yes.
I added support to change the MP size during job lunch, and for that I need to wrap it in `FullyShardedDataParallel` inside the `build_model`.
As I don't want to double wrap it, I needed to add this if..